Project.toml

```diff
@@ -4,23 +4,19 @@ authors = ["Jim Pivarski <pivarski@princeton.edu>", "Jerry Ling <jerry.ling@cern
 version = "0.1.2"
 
 [deps]
+Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
-Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
-
-[weakdeps]
 PythonCall = "6099a3de-0909-46bc-b1f4-468b9a2dfc0d"
-
-[extensions]
-AwkwardPythonCallExt = "PythonCall"
+Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
 JSON = "0.21.4"
-julia = "1.9"
 Tables = "1.11.1"
+julia = "1.9"
+PythonCall = "0.9"
 
 [extras]
-PythonCall = "6099a3de-0909-46bc-b1f4-468b9a2dfc0d"
```

Comment:
@Moelf - it could be due to my own setup, but I did not manage to get this `PythonCall` dependency when running from Python REPL. This is an attempt to fix it.

Reply:
before it was set up in a way such that installing `AwkwardArray.jl` does NOT imply needing a Python. If I were to do this, I would just make sure from the python side, it calls something to install `PythonCall`.

The way this PR implements it, you cannot use this package as a pure-Julia package, which may or may not be what you want.

Reply:
That's what we discussed today. Although it would be better (more opportunities) for this to be usable as a pure-Julia package, @ianna ran into some problems attempting to do so, and I think its primary value would be for users of both Python and Julia. (In Julia, there are already packages for some of these use-cases, such as ArrayOfArrays and StructArrays.)

So the decision between optionally depending on Python and strictly depending on Python is the result of a cost-benefit analysis. The benefit is small but positive; how big is the cost?

Reply:
the way this PR does it sounds reasonable to me.